instructors.md

```diff
@@ -199,6 +199,41 @@ A typical schedule for a two-day course is:
     *   15:45 Open discussion
     *   16:15 Wrap up and minute cards for afternoon session
 
+## Why Do(n't) We Teach X?
+
+Workshop attendees and trainee instructors often ask why we don't teach
+high-performance computing, machine learning, Perl, or a long list of other topics.
+Our answer is that as with every curriculum,
+the question is not, "What would we like to add?"
+but, "What are we willing to take out in order to make room?"
+We believe our core topics are the absolute minimum that researchers need to know
+in order to work efficiently and reproducibly.
+More importantly,
+we don't know what we could take out to make space for something else.
+
+One thing we *do* know is that we do not wish to become embroiled in debates
+over the relative merits of different languages or operating systems.
+No one has ever demonstrated that R programmers are more productive than Python programmers,
+and proficient users of Windows seem just as productive as equally-proficient users of Unix.
+If a learner asserts that their favorite tool is better than alternatives,
```

Comment:
also, perhaps suggest distinguishing between "better" and "better for me" to the assertor